packages/braid-design-system/color-mode/index.ts

```diff
@@ -6,8 +6,9 @@ const flag = '_bdsdm';
 // 2 = OS Preference
 // TODO: COLORMODE RELEASE
 // Finalise import contract
+/** @deprecate Use `import { colorModeQueryParamCheck } from 'braid-design-system/color-mode/query-param'` instead. */
```

Comment:
<img width="598" alt="with @deprecate" src="https://user-images.githubusercontent.com/3297808/214481798-d269dc73-eff3-4b09-9f8d-f3a0fd467ed0.png">


```suggestion
/** @deprecated Use `import { colorModeQueryParamCheck } from 'braid-design-system/color-mode/query-param'` instead. */
```
<img width="627" alt="with @deprecated" src="https://user-images.githubusercontent.com/3297808/214481809-bd9bc264-d7b4-4a56-82c7-985cef05cd0b.png">